src/assets/js/scope.js

```diff
@@ -28,6 +28,7 @@ class Scope {
     this.isStaff = isStaff;
     this.isNdaed =
       'nda' in (mozilliansorgGroups || {}) ||
```

Comment:
Could you please add a code comment pointing to the _other_ locations here, too, since we seem to have hard-coded this in so many places?

I will not be sad if you choose not to, and instead have each of the pull requests reference the others. That would be fine too instead. Since this will always be spelunking.